### ISSUE TITLE
"--remove-indel" Flag Removes Some SNPs Too

### ISSUE THREAD

**Issue by oryoruk** [vcftools/vcftools#78]
Following line from phaseIII release of 1000G variant data (chromosome 22) includes both a SNP and an INDEL. I would expect the "--remove-indel" flag to remove the INDEL keep the SNP. However, it remove both:

 22      16577044        rs374006257;rs200929253 TG      AG,T    100     PASS    AC=17,47;AF=0.00339457,0.00938498;AN=5008;NS=2504;DP=14269;EAS_AF=0.0149,0;AMR_AF=0,0.0014;AFR_AF=0,0.0348;EUR_AF=0.001,0;SAS_AF=0.001,0;VT=SNP,INDEL;MULTI_ALLELIC     GT

I can see how this is happening, and I suspect it can be fixed quickly.

**Comment by Cashalow** [comment]
I just encountered the same bug, ```--remove-indel``` removes such an entry too:

```
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	S1	S10	S2	S3	S4	S5	S6	S7	S8	S9
NC_003923	103786	.	CG	TT	10206.9	PASS	.	GT:QR:GL:AO:AD:QA:DP:RO	.:.:.:.:.:.:.:.	1:0:.,.,.:298:0,298:11350:299:0	.:.:.:.:.:.:.:.	.:.:.:.:.:.:.:.	.:.:.:.:.:.:.:.	.:.:.:.:.:.:.:.	.:.:.:.:.:.:.:.	.:.:.:.:.:.:.:.	.:.:.:.:.:.:.:.	.:.:.:.:.:.:.:.
```

although it is a double mutation. It still removes it REF = CG and ALT = CT, as in your example. 

It's quite a worrisome bug, @oryoruk did you have a feedback on your issue ?

**Comment by auton1** [comment]
I do not think this is a bug. To me, this isn't an indel, as no nucleotides are inserted or deleted. Rather this is a multi-nucleotide polymorphism.

In the VCF format, this variant could be equally well represented by two single nucleotide entries like so:
NC_003923	103786	.	C	T
NC_003923	103787	.	G	T

In this case, we wouldn't expect VCFtools to treat them as indels, so why would we want to do that if they are represented in the original way?

**Comment by auton1** [comment]
Edit: Oh - I see. I mis-read your comment. My bad. Yes, this should be fixed.

**Comment by Cashalow** [comment]
I am sorry I have just phased out vcftools of my pipeline. I had seen https://github.com/vcftools/vcftools/issues/77 and decided to change to another more supported tool. Moreover I am not that fluent at all in C...
I am afraid you are on your own if you wish to fix this problem. Sorry.
